browser-test/src/support/applicant_questions.ts

```diff
@@ -47,6 +47,39 @@ export class ApplicantQuestions {
     await this.validateInputValue(zip, '.cf-address-zip input')
   }
 
+  async checkDefaultAddressQuestionValue(
```

Comment:
To be more precise this method should say somehting like "checkAddressQuestionValueWithDefaultStateWashington" because this method checks that state is set to Washington. But I don't think that having it as a separate method warranted. You can use "checkAddressQeustionValue" and pass state as "Washington" to it and add a comment there saying that it verifires that default state is shown. Like

```

      await applicantQuestions.checkDefaultAddressQuestionValue(
        '1234 St',
        'Unit B',
        'Sim',
        // Should be set to default state
        'Washington', 
        '54321',
      )
```

Reply:
oh nice call! will change!